docs/import-api-definition.md

```diff
@@ -0,0 +1,38 @@
+# Import API definition to 3scale
+
+When you import an API defintion, the following actions will be performed in the background:
```

Comment:
I would add another line around the spirit of the import feature

Using an API Definition format like OpenAPI or RAML you can automate some setups and save a lot of time in configuration.